Sources/MongoSwift/MongoDatabase.swift

```diff
@@ -129,11 +140,17 @@ public class MongoDatabase {
         self._database = nil
     }
 
-    /// Drops this database.
-    /// - Throws:
-    ///   - `ServerError.commandError` if an error occurs that prevents the command from executing.
-    public func drop(session: ClientSession? = nil) throws {
-        let operation = DropDatabaseOperation(database: self, session: session)
+    /**
```

Comment:
```suggestion
   /**
```
same alignment thing